ui/app/src/views/PegAssetPage.vue

```diff
@@ -110,7 +110,7 @@ export default defineComponent({
 
     const nextStepAllowed = computed(() => {
       const amountNum = new BigNumber(amount.value);
-      const balance = accountBalance.value?.toFixed(18) ?? "0.0";
+      const balance = accountBalance.value?.toFixed() ?? "0.0";
```

Comment:
@59023g so here we let the Asset.decimals decide how many decimals to show, right? And this is only for displaying. Before we send it to the backend, we have to add missing decimals for assets with decimals < 18? (e.g. fill missing decimals with 0, essentially the opposite of `trimZeroes` that we have in `swapCalculator`).

Reply:
`nextStepAllowed` returns a boolean, which decides if user can click the submit button to Peg or not. So this `balance` variable is limited in scope to `nextStepAllowed` as its used to decide truthiness. Does not go to display beyond said truthiness.
<img width="421" alt="Screen Shot 2021-02-10 at 12 34 45 AM" src="https://user-images.githubusercontent.com/5235538/107474351-cfe1ff00-6b37-11eb-979e-061eae37d764.png">



As far as "Before we send it to the backend, we have to add missing decimals for assets with decimals < 18" - I don't know. I did test peg function while changing decimal places to < 18. It appeared to work.